src/apps/chifra/internal/receipts/output.go

```diff
@@ -104,13 +104,14 @@ func (opts *ReceiptsOptions) ReceiptsInternal() (err error, handled bool) {
 	}
 
 	err = output.StreamMany(ctx, opts.Globals.Writer, fetchTransactionData, output.OutputOptions{
```

Comment:
The way the fix to Globals is working out, I think this will be the only line with a conflict. I'm going to merge this change first into `develop` and then merge into my code for the correction to Globals. Easier.